src/guidellm/benchmark/progress.py

```diff
@@ -682,7 +661,7 @@ def _update_processing_states(
 
     def _update_request_stats(
         self,
-        request_concurrency: int | None = None,
+        request_concurrency: float | None = None,
```

Comment:
Is this a float because it's a statistic of average integer concurrency over a period?